app/assets/javascripts/controllers/schedule/schedule_form_controller.js

```diff
@@ -59,7 +59,7 @@ ManageIQ.angular.app.controller('scheduleFormController', ['$http', '$scope', 's
         $scope.scheduleModel.name         = data.schedule_name;
         $scope.scheduleModel.timer_typ    = data.schedule_timer_type;
         $scope.scheduleModel.timer_value  = data.schedule_timer_value;
-        $scope.scheduleModel.start_date   = data.schedule_start_date;
+        $scope.scheduleModel.start_date   = moment.utc(data.schedule_start_date, 'MM/DD/YYYY').toDate()
```

Comment:
Not sure this is right.. we're parsing the date as UTC but what happens if `data.schedule_time_zone` is *not* UTC?